train.py

```diff
@@ -0,0 +1,52 @@
+import argparse
+import re
+import yaml
+import os
+import hydra
+from hydra.core.config_store import ConfigStore
+from pathlib import Path
+
+
+# Hack to fix multiprocessing deadlock when PyTorch's DataLoader is used
+# (more info: https://github.com/pytorch/pytorch/issues/1355)
+import cv2
+cv2.setNumThreads(0)
+
+# from src.constructor import TASKS
+from src.constructor.config_structure import ConfigParams
+# from src.constructor import create_trainer
+
+
+@hydra.main()
+def load_config(config):
+    # Need to add --config-path (-cp) and --config_name (-cn) in run command
+    # Example config in configs/classification_cifar10.yaml
+    # Then the command line will be:
+    # python --config-path configs --config_name classification_cifar10 
+    config_params = ConfigParams(**config)
+    return config_params
+
+if __name__ == '__main__':
```

Comment:
Can the config parameters be overriden by cli arguments?

Reply:
Yes hydra support change any config parameters by use cli

Reply:
Actually no =). I need to remove argparser to give hydra ability change config by cli

Reply:
Ok, let's do it then